src/site/content/en/blog/referrer-best-practices/index.md

```diff
@@ -285,34 +278,40 @@ app.use(helmet.referrerPolicy({policy: 'strict-origin-when-cross-origin'}));
 
 ### What if `strict-origin-when-cross-origin` (or stricter) doesn't accommodate all your use cases?
 
-In this case, don't set an unsafe policy such as `unsafe-url`. What you can do instead is take a
-**progressive approach**: set a protective policy for your website and if need be, a more permissive
-policy for specific requests or elements.
+In this case, take a **progressive approach**: set a protective policy like
+`strict-origin-when-cross-origin` for your website and if need be, a more permissive policy for
+specific requests or HTML elements.
 
-**Example:**
+### Example: element-level policy
 
 `index.html`:
 
-```html
-<meta name="referrer" content="strict-origin-when-cross-origin" />
-<img src="…" referrerpolicy="no-referrer-when-downgrade" />
+```html/6
+<head>
+  <!-- document-level policy: strict-origin-when-cross-origin -->
+  <meta name="referrer" content="strict-origin-when-cross-origin" />
+  <head>
+    <body>
+      <!-- policy on this <a> element: no-referrer-when-downgrade -->
+      <a src="..." href="..." referrerpolicy="no-referrer-when-downgrade"></a>
```

Comment:
Using the ellipsis character rather than 3 dots so that screen readers announce it as intended

Reply:
```suggestion
      <a src="…" href="…" referrerpolicy="no-referrer-when-downgrade"></a>
```